reporter-web-app/src/reducers/selectors.js

```diff
@@ -205,6 +205,7 @@ export const getTableViewProjectFilterSelections = memoizeOne(
         const webAppOrtResult = getOrtResult(state);
```

Comment:
commit message: 
'hte' -> 'the'
remove double space after 'drop-down'
a.text is one time in ticks, one time it is not.